notification-support/src/main/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventUtils.java

```diff
@@ -51,6 +52,18 @@ public static AvroKey toAvroKeyRecord(Schema schema) {
         .build();
   }
 
+  public static AvroKey toAvroKeyRecord(SchemaKey schemaKey) {
```

Comment:
is it _ever_ valid for the avro key to be null? Or should we throw an exception / not persist.
I don't know the answer to that.

Reply:
No, it should never be null. Can we avoid the `Optional` here?

Reply:
Sure, since I added new field I didn't want to break the schema compatibility. I assumed it is currently `BACKWARD` compatible.

That's why I used `Optional.ofNullable`

Reply:
Removed